nixos/modules/misc/version.nix

```diff
@@ -95,6 +95,7 @@ in
         PRETTY_NAME="NixOS ${cfg.version} (${cfg.codeName})"
         LOGO="nix-snowflake"
         HOME_URL="https://nixos.org/"
+        DOCUMENTATION_URL="https://nixos.org/nixos/manual/index.html"
```

Comment:
This is new in v240: https://github.com/systemd/systemd/blob/f2ccf8320ac5d121cc8f50622ae3c9af1d0bfea2/NEWS#L369